openstudio-extension.gemspec

```
@@ -26,10 +26,10 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '~> 2.5.0'
+  spec.required_ruby_version = '~> 2.7.0'
 
   spec.add_dependency 'bcl', '~> 0.6.1'
-  spec.add_dependency 'bundler', '~> 2.1'
+  spec.add_dependency 'bundler', '~> 2.2'
   spec.add_dependency 'octokit', '~> 4.18.0' # for change logs
```

```suggestion
  spec.add_development_dependency 'octokit', '~> 4.18.0' # for change logs
```

If this is only for changelog, this should be a dev dependency

---

Just to make sure, dependent gems will install this development dependency? I just want to make sure that it is just not a dev dependency for this gem.

---

https://guides.rubygems.org/specification-reference/#add_development_dependency

> Development dependencies aren't installed by default and aren't activated when a gem is required.